Resources/Prototypes/Entities/Objects/Fun/crayons.yml

```diff
@@ -88,13 +88,21 @@
   - type: Crayon
     color: Red
     selectableColor: true
-    capacity: 30
+    capacity: 1000000 # Delta V - Let there be art!
```

Comment:
I highly suggest making this a more reasonable value. The functional difference between 1,0000,000 and something like 100 is not that large, but prevents abuse. Also, crayons do get used up irl - maybe add in a new research for bluespace crayons that have infinite uses if you really want that.

Reply:
I am being honest- I will probably try and figure out how to port later and instead add the magic crayon PR that was thrown at me in the above comments